lib/fswatcher/errors_linux.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package fswatcher
+
+import (
+	"syscall"
+)
+
+func interpretNotifyWatchError(err error, folder string) error {
```

Comment:
I'd typically expect this as an `isWatchesTooFew(err error) bool` that could be used appropriately by someone inspecting a returned error, instead of something that transforms the error, cf. `os.IsNotExist` and similar.